### PR TITLE
MMTF write support

### PR DESCRIPTION
**Pull request by frodofine**
The title is self-explanatory.

Addresses #149

### CODE REVIEW

File: include/chemfiles/formats/MMTF.hpp [hunk 1]
```diff
@@ -21,20 +21,24 @@ class MMTFFormat final: public Format {
...
     void read(Frame& frame) override;
+    void write(const Frame& frame) override;
     size_t nsteps() override;
 
 private:
 
     mmtf::StructureData structure_;
 
+    std::string filename_;
+    size_t writeAtomLoc_ = 0;
```
Comment (Luthaf) [inline include/chemfiles/formats/MMTF.hpp]: What does is this used for ? It seems to be used as a flag to know whether to write or not in the destructor, but the variable name seems to indicate something else.

**Comment by frodofine** [comment]
> So, correct me if I am wrong, but this code is only writing to the file in the destructor, and is adding new models to the structure in write, right?

Yes, `mmtf::compressGroupList(structure_);` can only be called once after all the models have been added.

**Comment by frodofine** [comment]
Unless there are any major concerns, I'm going to merge this. I have some more features, but I think I'll include them in #185.

File: include/chemfiles/formats/MMTF.hpp (hunk 1 above)
Comment (Luthaf) [reply include/chemfiles/formats/MMTF.hpp]: Could you add a bit of doc for this and the other members ?

Comment (frodofine) [reply include/chemfiles/formats/MMTF.hpp]: Sure thing.

**Comment by Luthaf** [comment]
Sounds good to me ! Feel free to merge when CI passes =)

**Pull request merged**